ArkKitTests/ark-rule-kit-tests/ArkRuleKitTests.swift

```diff
@@ -39,59 +39,59 @@ class ArkRuleKitTests: XCTestCase {
     }
 
     /// INTEGRATION TESTS
-    func testForeverExecution_shouldExecuteTwice() {
-        let eventManager = ArkEventManager()
-        var executedEvents: [EventStub] = []
-
-        let mockCallback: ActionCallback<EventStub> = { event, _ in
-            executedEvents.append(event)
-        }
-
-        let forever = Forever(mockCallback)
-        var eventStub = EventStub()
-        let actionContext = ArkActionContext(ecs: MockECSContext(),
-                                             events: eventManager,
-                                             display: MockDisplayContext(),
-                                             audio: MockAudioContext())
-        eventManager.subscribe(to: EventStub.id, { event in
-            guard let event = event as? EventStub else {
-                return
-            }
-            forever.execute(event, context: actionContext)
-        })
-        eventManager.emit(eventStub)
-        eventManager.processEvents()
-        XCTAssertEqual(executedEvents.count, 1, "Callback should be called")
-        eventManager.emit(eventStub)
-        eventManager.processEvents()
-        XCTAssertEqual(executedEvents.count, 2, "Callback should be called")
-    }
-
-    func testOnceExecution_shouldExecuteOnce() {
-        let eventManager = ArkEventManager()
-        var executedEvents: [EventStub] = []
-
-        let mockCallback: ActionCallback<EventStub> = { event, _ in
-            executedEvents.append(event)
-        }
-
-        let once = Once(mockCallback)
-        var eventStub = EventStub()
-        let actionContext = ArkActionContext(ecs: MockECSContext(),
-                                             events: eventManager,
-                                             display: MockDisplayContext(),
-                                             audio: MockAudioContext())
-        eventManager.subscribe(to: EventStub.id, { event in
-            guard let event = event as? EventStub else {
-                return
-            }
-            once.execute(event, context: actionContext)
-        })
-        eventManager.emit(eventStub)
-        eventManager.processEvents()
-        XCTAssertEqual(executedEvents.count, 1, "Callback should be called")
-        eventManager.emit(eventStub)
-        eventManager.processEvents()
-        XCTAssertEqual(executedEvents.count, 1, "Callback should be called")
-    }
+//    func testForeverExecution_shouldExecuteTwice() {
```

Comment:
@ryanpeh for your fixing

Reply:
oh sorry, that's on me. i commented out test cases for `Once` and `Forever` since we've removed them